sentinel-adapter/sentinel-api-gateway-adapter-common/src/main/java/com/alibaba/csp/sentinel/adapter/gateway/common/slot/GatewaySlotChainBuilder.java

```diff
@@ -44,8 +44,8 @@ public ProcessorSlotChain build() {
         chain.addLast(new LogSlot());
         chain.addLast(new StatisticSlot());
         // Rule checking slot
-        chain.addLast(new AuthoritySlot());
         chain.addLast(new SystemSlot());
+        chain.addLast(new AuthoritySlot());
```

Comment:
In fact, the order of the slots in the `GatewaySlotChainBuilder` is more appropriate. Maybe we could update the `DefaultSlotChainBuilder`.

Reply:
Okay, it's been corrected.